test/src/test/java/org/corfudb/integration/MemoryFootprintIT.java

```diff
@@ -28,6 +29,15 @@
     private static int corfuStringNodePort;
     private static String singleNodeEndpoint;
 
+    private CorfuRuntime runtime;
```

Comment:
Would it make more sense to move this repeated code to AbstractIT?

Reply:
Yes, I'm trying to move it to the cleanUp() method in AbstractIT.

Reply:
Done.